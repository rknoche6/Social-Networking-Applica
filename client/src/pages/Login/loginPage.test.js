import React from "react";
import { shallow } from "enzyme";
import LogInPage from "./loginPage";

describe("LogInPage", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<LogInPage />)));

  it("should render correctly", () => expect(wrapper).toMatchSnapshot());

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(1);
  });
});