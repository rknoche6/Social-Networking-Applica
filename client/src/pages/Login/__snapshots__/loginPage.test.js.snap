// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogInPage should render correctly 1`] = `ShallowWrapper {}`;